configs/webpack.config.web.dev.babel.js

```diff
@@ -174,7 +174,8 @@ export default merge(commonConfig, {
 
   devServer: {
     host: HOST,
-    allowedHosts: 'all',
+    // allowedHosts: 'all',
```

Comment:
In my opinion, will be better remove commented code

Reply:
removed